server/modules/slack/slackIntegration.js

```diff
@@ -160,3 +168,27 @@ export const addToChannel = async (fellowEmail, channel) => addOrRemove(fellowEm
  * @returns {Object} - The result after a user has been removed from a channel
  */
 export const removeFromChannel = async (fellowEmail, channel) => addOrRemove(fellowEmail, channel, 'kick');
+
+/**
+ * @function onboardingAutomation
+ * @desc combines all onboarding functions
+ *       into one function.
+ *
+ * @param {string} fellowEmail - user's email
+ * @param {string} channel - the channel id
+ * @param {string} partnerId - the prtner id
+ *
+ * @returns {Object} - With a success message
+ */
+
+export const onboardingAutomation = async (fellowEmail, channel, partnerId) => {
+  try {
+    removeFromChannel(fellowEmail, channel);
```

Comment:
Looking at the implementation of `removeFromChannel` function, there is a possibility that it would return an *ERROR OBJECT*, I don't think that is properly handled in here.

Reply:
I have added a conditional to check if the response from this call is an error and returned an error message